package.json

```diff
@@ -99,7 +99,7 @@
     "core-js": "3.6.5",
     "flexboxgrid": "6.3.1",
     "focus-trap": "5.1.0",
-    "fundamental-styles": "0.11.0",
+    "fundamental-styles": "0.12.0-rc.55",
```

Comment:
Let's revert this change and include css code from `object-marker.css` into `object-marker.component.scss`

Reply:
This PR will merged post the Breaking changes are fixed if i revert this change build will fail. this is already conveyed to @InnaAtanasova